.changeset/silly-moles-rush.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/firebase": minor
```

Comment:
I didn't catch this in the previous PR but the package name is `firebase`, there's no `@firebase/firebase`. Also I think these should be patch because there's no API change.